circuits-circom/circuits/garanti/test/garanti_registration.spec.ts

```diff
@@ -224,6 +211,20 @@ describe("Garanti registration WASM tester", function () {
         assert.equal(JSON.stringify(poseidon.F.e(hashed_onramper_id)), JSON.stringify(expected_hash), true);
     });
 
+    it.skip("Should fail if padding is not all zeroes", async () => {
```

Comment:
@richardliang See if we can get these tests to work, otherwise we skip them for now. I have tested it by manually generating witness and confirming it fails.